### PR TITLE
Adding Csharp photoeditor sample

### PR DESCRIPTION
**Pull request by bgursahani**
## Description

Please include a summary of the change and/or which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Checklist

- [ ] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [ ] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release). 
- [ ] Samples set the minimum supported OS version to Windows 10 version 1809.
- [ ] Samples build clean with no warnings or errors.
- [ ] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).

### CODE REVIEW

File: Samples/PhotoEditor/cs-winui/App.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            WindowHandle = WinRT.Interop.WindowNative.GetWindowHandle(Window);
+        }
+        public static MainWindow Window { get; private set; }
+        public static IntPtr WindowHandle { get; private set; }
+    }
+}
+
+
+
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/App.xaml.cs:42]: remove theses extra lines

File: Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+                base.OnNavigatingFrom(e);
+            }
+        }
+
+        /// <summary>
+        /// Gives users a chance to save the image before navigating
+        /// to a different page.
+        /// </summary>
+
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs:118]: remove empty lines

File: Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+                    WinRT.Interop.InitializeWithWindow.Initialize(fileSavePicker, App.WindowHandle);
+
+                    var outputFile = await fileSavePicker.PickSaveFileAsync();
+
+                    if (outputFile != null)
+                    {
+                        using (IRandomAccessStream outStream = await outputFile.OpenAsync(FileAccessMode.ReadWrite))
+                        {
+                            await offscreen.SaveAsync(outStream, CanvasBitmapFileFormat.Jpeg);
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs:232]: remove empty line

File: Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Foundation;
+using Windows.Graphics.Effects;
+using Microsoft.UI.Composition;
+using Windows.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Windows.Storage.Streams;
+using Microsoft.UI.Xaml;
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs:19]: one line between using statements and `namespace... {`

File: Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+        {
+            var brush = (LoadedImageBrush)d;
+            // Unbox and set a new blur amount if the CompositionBrush exists.
+            brush.CompositionBrush?.Properties.InsertScalar("Blur.BlurAmount", (float)(double)e.NewValue);
+        }
+
+
+
+        public static readonly DependencyProperty ContrastAmountProperty = DependencyProperty.Register(
+"ContrastAmount",
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs:78]: formatting/indentation is off here

File: Samples/PhotoEditor/cs-winui/Package.appxmanifest [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="8d1fcdb6-2142-4f1d-8a99-cf03513c7298"
+    Publisher="CN=bgursahani"
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/Package.appxmanifest:11]: change to Microsoft Corporation - see https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/Windowing/cs-winui/Package.appxmanifest as an example

**Review by stevewhims — APPROVED** [review]

File: Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+            set { SetValue(ExposureAmountProperty, value); }
+        }
+
+        private static void OnExposureAmountChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var brush = (LoadedImageBrush)d;
+            // Unbox and set a new blur amount if the CompositionBrush exists.
+            brush.CompositionBrush?.Properties.InsertScalar("ExposureEffect.Exposure", (float)(double)e.NewValue);
+        }
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/LoadedImageBrush.cs:103]: extra line

File: Samples/PhotoEditor/cs-winui/MainWindow.xaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    
+    <Window
+    x:Class="PhotoEditor.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PhotoEditor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/MainWindow.xaml:12]: extra line

File: Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+            if (replaceImage == true)
+            {
+                MainPage.Current.Images.RemoveAt(index);
+                MainPage.Current.Images.Insert(index, item);
+            }
+            else
+            {
+                MainPage.Current.Images.Insert(index + 1, item);
+            }
+
```
Comment (angelazhangmsft) [inline Samples/PhotoEditor/cs-winui/DetailPage.xaml.cs:282]: extra line

**Review by angelazhangmsft — APPROVED** [review]
Just a few extra formatting comments, otherwise looks good to me :)

**Pull request merged**